extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -23,15 +23,17 @@ import {closestByTag, isRTL, tryFocus} from '../../../src/dom';
 import {createCustomEvent} from '../../../src/event-helper';
 import {debounce} from '../../../src/utils/rate-limit';
 import {dev} from '../../../src/log';
-import {parseUrlDeprecated, removeFragment} from '../../../src/url';
+import {removeFragment} from '../../../src/url';
 import {setStyles, toggle} from '../../../src/style';
 import {toArray} from '../../../src/types';
-/** @const */
+
+/** @private @const {string} */
```

Comment:
nit: The annotation changes are unnecesary since Closure can infer from the declaration that these are `@const` and `string` or `number` primitives. Did the linter complain about these?

Reply:
Just for completion.